readthedocs/config/config.py

```diff
@@ -451,11 +444,14 @@ def validate_conda(self):
                     self.PYTHON_INVALID_MESSAGE,
```

Comment:
I think this message is wrong. It will say

    "python" section must be a mapping.

but we are under `conda` section.

Reply:
Yeah, this was ported from the previous config file, I'm not sure if this is in the scope of this PR, but now we have a `validate_dict` function for this cases.

Reply:
If it's a problem, and won't be fixed in this PR, please open an issue for this so we don't forget to fix it.

Reply:
I opened https://github.com/rtfd/readthedocs.org/issues/4530 for this